micro_brewery/data/pos_config.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="point_of_sale.pos_config_main" model="pos.config">
+        <field name="start_category" eval="1"/>
+        <field name="limit_categories" eval="1"/>
+        <field name="iface_available_categ_ids" eval="[(6, 0, [ref('pos_category_3')])]"/>
+        <field name="iface_start_categ_id" ref="pos_category_3"/>
+    </record>
+    <record id="pos_config_main_bar_micro_brewery" model="pos.config">
+        <field name="name">Bar</field>
+        <field name="warehouse_id" ref="stock.warehouse0"/>
+        <field name="iface_available_categ_ids" eval="[(6, 0, [ref('pos_category_2')])]"/>
+        <field name="limit_categories" eval="True"/>
+        <field name="iface_splitbill" eval="True"/>
+        <field name="floor_ids" eval="[(6, 0, [ref('restaurant_floor_1'), ref('restaurant_floor_2')])]"/>
+    </record>
+    <record id="pos_restaurant.pos_config_main_restaurant" model="pos.config">
+        <field name="active" eval="False"/>
+    </record>
```

Comment:
And we'll also avoid to archive existing ones in data, and only archive in demo.